src/clusto/services/http.py

```diff
@@ -93,9 +93,22 @@ def setattr(self, request):
         Set an attribute on this object.
 
         Requires HTTP parameters "key" and "value"
-        Optional parameters are "subkey" and "number"
+        Optional parameters are "subkey", "number" and "datatype"
         '''
         kwargs = dict(request.params.items())
+        datatype = kwargs.get('datatype', None)
+        kwargs.pop('datatype', None)
+
+        if datatype == 'relation':
+            try:
+                kwargs['value'] = clusto.get_by_name(kwargs['value'].rsplit('/', 1)[1])
+            except LookupError:
+                return Response(status=404, body='404 Not Found\n')
```

Comment:
I think this should return a plain 400, instead of 404: the supplied element couldn't be found, put that in the body of the error